src/content/docs/projects/ToPWR/Communication Channels.md

```diff
@@ -0,0 +1,90 @@
+  - Spotkania (np. Weekly, spontaniczne).
+  - Uzgodnienia dotyczące designu, kodu, dokumentacji.
+
+- **Messenger**:
+
+  - Szybkie informacje i ogłoszenia.
+  - Natychmiastowe odpowiedzi, sprawy „na już”.
+  - Miękkie sprawy organizacyjne (np. potwierdzenia, dogadywanie terminów).
+
+- **GitHub**:
+  - Rozmowy o konkretnych fragmentach kodu.
+  - Pull Requesty, Issues, dokumentowanie zmian.
+  - Miejsce na krytyczne decyzje dotyczące kodu.
+
+> **Notka**: Powyższy podział nie jest sztywny. Zespół może decydować o elastycznym korzystaniu z dowolnego kanału. **Jednakże**, proponowany podział jest mocno zalecany. Musi być on jednolity dla całego programu ToPWR. Najważniejsze, by ustalenia, które będą istotne w przyszłości, zawsze były odpowiednio udokumentowane i łatwe do odnalezienia.
+
+## Kanały komunikacyjne
+
+### Discord
+
+**Cel**:
+
+- „Twarde” sprawy techniczne: dyskusje o designie, ustalenia kodu, architektury, dokumentacji itp.
+- Dłuższe wątki rozmów, które wymagają wnikliwej analizy czy uzgodnień.
+- Organizowanie spotkań Weekly, spontanicznych spotkań ASAP.
+
+**Zalecenia**:
+
+1. **Wydzielone kanały tematyczne**: Aby uniknąć chaosu, korzystaj z kanałów na Discordzie zgodnie z ich przeznaczeniem (np. #backend, #mobile, #main, etc.).
+2. **Dokumentuj ustalenia**: Po zakończeniu ważnej dyskusji, kluczowe wnioski przenieś do odpowiedniego miejsca (np. do issue na GitHub albo Google Docs).
+3. **Używaj wątków**: Pamiętaj o używaniu wątków do dłuższych rozmów (threads).
+
+### Messenger
+
+**Cel**:
+
+- Szybkie informacje lub ogólne ogłoszenia.
+- Sytuacje wymagające błyskawicznej reakcji (ASAP).
+- „Miękkie” sprawy organizacyjne (dogadywanie terminów spotkań, potwierdzenia, krótkie pytania).
+
+**Zalecenia**:
+
+1. **Proste pytania i szybkie decyzje**: Jeżeli potrzebujesz natychmiastowej odpowiedzi — Messenger jest najlepszy.
+2. **Nie nadużywaj**: Częste powiadomienia na Messengerze mogą rozpraszać. Jeśli sprawa nie jest pilna, rozważ użycie Discord.
+3. **Potwierdzaj kluczowe ustalenia**: Po zakończeniu rozmowy w Messengerze, która ma wpływ na projekt, zapisz krótkie podsumowanie np. w notatce lub w innym, bardziej „trwałym” kanale (np. Discord/Google Docs). Możesz też odezwać się do jednego z kierowników projektu.
+
+### GitHub (Komentarze/Pull Requesty)
+
+**Cel**:
+
+- Dyskusje i uwagi na temat konkretnych fragmentów kodu.
+- Omawianie pull requestów i dokumentowanie zmian w kodzie.
+- Zgłaszanie i rozwiązywanie issues.
+
+**Zalecenia**:
+
+1. **Konkretne komentarze**: Komentuj bezpośrednio w kodzie przy danym fragmencie (np. w PR), aby wiadomo było, do czego odnosisz swoją uwagę.
+2. **Opisowe PR**: Twórz pull requesty z opisem kontekstu, co i dlaczego zostało zmienione oraz jak przetestować te zmiany.
+3. **Używaj szablonów**: Jeśli repozytorium twojego projektu posiada szablony PR lub issue, wypełniaj je, aby zachować spójny format w całym zespole.
+
+## Sugerowane dobre praktyki i dodatkowe pomysły dla kierowników
```

Comment:
rozumiem idee ale czuje pewien dysonans że handbook jest niby do ToPWR a jednocześnie ogólny dla PMów

Reply:
Nie wiem co z tym faktem zrobimy

Reply:
Myślę, że to nie jest problem. Guide jest dosyć mocno kierowany pod nasze praktyki, a dodanie tu takich ogólników (moim zdaniem) jednak go wzbogaca. Nie przejmowałbym się B))